example/src/child-component/index.js

```diff
@@ -7,7 +7,7 @@ import computed from "./computed";
 
 export const addState = withState({ effects: effectDefs, initialState, computed });
 
-export const ChildComponent = (props, { state, effects }) => (
+export const ChildComponent = ({ state, effects }) => (
```

Comment:
Do you think it would make sense to spread both? Or keep as keyed props? I'm on the fence re: ease of access vs collision potential